pkgs/top-level/python-packages.nix

```diff
@@ -1178,6 +1178,45 @@ in {
     };
   };
 
+  couchpotato = buildPythonPackage rec {
```

Comment:
couchpotato is used as an application and not as a library. Therefore, it doesn't belong in `python-packages.nix`. Instead, it should fit somewhere else in Nixpkgs tree based on what it is/does.